src/pyfmi/fmi.pyx

```diff
@@ -1190,13 +1190,14 @@ cdef class FMUModelBase(ModelBase):
     """
     An FMI Model loaded from a DLL.
     """
-    def __init__(self, fmu, path='.', enable_logging=None, log_file_name="", log_level=FMI_DEFAULT_LOG_LEVEL, _unzipped_dir=None, _connect_dll=True):
+    def __init__(self, fmu, path='.', enable_logging=None, log_file_name="", log_level=FMI_DEFAULT_LOG_LEVEL,
+                 _unzipped_dir=None, _connect_dll=True, allow_unzipped_fmu = False):
         """
```

Comment:
We are missing a good docstring on this class, lets take the opportunity to add it.

Reply:
Please review:
https://github.com/modelon-community/PyFMI/commit/09901c64e809fa707c952d070c5381fe24a632e7